pkg/tracing/jaeger/jaeger.go

```diff
@@ -7,6 +7,7 @@ import (
 	"context"
 	"fmt"
 	"io"
+    "strings"
```

Comment:
This file seems to be incorrectly formatted. This line is indented with spaces whereas all of the others are indented with tabs. can you pls run gofmt?

Reply:
Oops, fixed.